src/pkg/runner/runner.go

```diff
@@ -175,6 +187,63 @@ func (r *Runner) importTasks(includes []map[string]string, dir string, setVariab
 	return nil
 }
 
+func (r *Runner) loadIncludedTaskFile(taskName string) (string, error) {
```

Comment:
Does this fn actually "load" the included task file or does it just list the tasks?

Reply:
EDIT: disregard, my bad, was looking at a different file